api/v1alpha1/packagebundle.go

```diff
@@ -67,12 +69,34 @@ func (config *PackageBundle) GetMajorMinorBuild() (major int, minor int, build i
 	return major, minor, build
 }
 
```

Comment:
It looks like this could be a private method and maybe the other method could call it? I'm okay testing private methods

Reply:
I made the method to private and created an export file to finish unit testing (open to other suggestions).